src/Features/CSharp/Portable/Completion/CompletionProviders/KeywordCompletionProvider.cs

```diff
@@ -97,6 +97,7 @@ private static ImmutableArray<IKeywordRecommender<CSharpSyntaxContext>> GetKeywo
                 new NameOfKeywordRecommender(),
                 new NamespaceKeywordRecommender(),
                 new NewKeywordRecommender(),
+                new NotNullKeywordRecommender(),
```

Comment:
Is there a test for this?

Reply:
> Is there a test for this?

What kind of test do you  have in mind?

Reply:
~~Check if `notnull` shows as keyword in completion list~~

Ah, I see I have missed the comments above. Please ignore this